lk2nd/device/dts/msm8974/victara.dts

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
I'd appreciate if you license new dt submissions as `BSD-3-Clause` in line with upstream linux preference